src/devices/somfy_iohc.c

```diff
@@ -91,60 +91,123 @@ Example packets:
 
 #include "decoder.h"
 
+struct iohc_msg {
```

Comment:
Do you really need a struct? I.e. using just e.g. `iohc_end_flag` or `msg_end_flag` would be simpler to maintain.
Otherwise you will need to put a `@fn` on the doc-comment above.

Reply:
Using a struct is both clean and easy to initialize to zero. There's no way it would add maintenance burden per se, only if you insist on adding documentation. As you'd accept the struct members as standalone variables without documentation, adding it to the struct seems rather pointless. It's not in a shared header file anyway.